images/network-controller/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.svc.ci.openshift.org/openshift/builder:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/sdn
+COPY . .
+RUN make build --warn-undefined-variables WHAT=cmd/network-controller
```

Comment:
WHAT isn't supported
I think there is a separate target for every binary hopefully, try `make help`

Reply:
k